core/src/server/access_control/host.rs

```diff
@@ -127,7 +127,7 @@ impl Host {
 }
 
 impl Pattern for Host {
-	fn matches<T: AsRef<str>>(&self, other: T) -> bool {
+	fn matches<'a, T: AsRef<str> + PartialEq<&'a str>>(&self, other: T) -> bool {
```

Comment:
I don't follow why this additional lifetime and `PartialEq` trait bound was added?

Reply:
This was added for checking the `OriginType` against "null" str, but could be removed